content/pages/FAQ.rst

```diff
@@ -21,8 +21,11 @@ Lightning talks are five minute presentations on any topic. At barcamp,
 the first session will include lightning talks in the barcamp style;
 anyone can sign up beforehand and present on any topic for up to
 five minutes instead of the full hour. We will still have full-length
-barcamp talks available during that session, but lightning talks will
-be an option.
+barcamp talks available in other rooms at that time. We only accept .pdf
```

Comment:
It should be a single exclusive session. I'm not sure when we usually have it but probably in early afternoon?